### PR TITLE
fix(avoidance): avoidance shift line processing bug

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Description

https://github.com/autowarefoundation/autoware.universe/pull/4731

## Tests performed

- [x] PASS TIER IV INTERNAL SCENARIOS

## Effects on system behavior

Improve avoidance performance.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by shmpwk — APPROVED** [review]

File: autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml [hunk 1]
```diff
@@ -4,8 +4,6 @@
     avoidance:
       resample_interval_for_planning: 0.3               # [m]
       resample_interval_for_output: 4.0                 # [m]
-      detection_area_right_expand_dist: 0.0             # [m]
```
Comment (kyoichi-sugahara) [inline autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:7]: I checked this change is applied in the parameter file in the [file](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/config/avoidance/avoidance.param.yaml)
but is related to this [PR](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/config/avoidance/avoidance.param.yaml)?

**Review by kyoichi-sugahara — APPROVED** [review]
LGTM but just curious about this change :thinking:

File: autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml (hunk 1 above)
Comment (satoshi-ota) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:7]: @kyoichi-sugahara Thanks for your checking.
I'm sorry fot the confusion. This diff is not related to https://github.com/autowarefoundation/autoware.universe/pull/4731 but is related to https://github.com/autowarefoundation/autoware.universe/pull/4525.

I forgot to remove unused params, and remove them in this PR togather.

Comment (kyoichi-sugahara) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:7]: Ok i see :+1: 
Thanks!

**Pull request merged**